apps/asap-server/src/controllers/calendars.ts

```diff
@@ -186,9 +187,12 @@ export interface CalendarController {
   ) => Promise<CalendarResponse>;
 
   fetchById(id: string, options?: { raw: false }): Promise<CalendarResponse>;
-  fetchById(id: string, options?: { raw: true }): Promise<CalendarRaw>;
+  fetchById(id: string, options?: { raw: true }): Promise<CalendarRawResponse>;
 }
 
 export type CalendarRaw = Calendar & {
   id: string;
 };
+export type CalendarRawResponse = CalendarRaw & {
```

Comment:
i think creating the new type is a little confusing - plus using the `Response` suffix indicates this would be, like the rest of the methods, meant for our API 

Simply add the version to the `CalendarRaw` type

Reply:
The function is already overloaded and returns 2 different types:

``` 
fetchById(id: string, options?: { raw: false }): Promise<CalendarResponse>;
fetchById(id: string, options?: { raw: true }): Promise<CalendarRaw>;
```

The other type has a `Response` suffix. The 2 are inconsistent.

Reply:
We should change `CalendarResponse`. 

`CalendarRawResponse` removed here: https://github.com/yldio/asap-hub/pull/1153/commits/cb6ad57c40dbdb9675e07c2ba35edb44ec10d571